emspy/query/localdata.py

```diff
@@ -82,14 +86,18 @@ def delete_data(self, table_name, condition = None):
 
 	def delete_all_tables(self):
 
-		for table_name in list(LocalData.table_info.keys()):
-			if self.table_exists(table_name):
-				self._conn.execute("DROP TABLE %s" % table_name)
-		self._conn.commit()
+		if self.__dbfile:
+			for table_name in list(LocalData.table_info.keys()):
+				if self.table_exists(table_name):
+					self._conn.execute("DROP TABLE %s" % table_name)
+			self._conn.commit()
 
 
 	def table_exists(self, table_name):
 
+		if self.__dbfile == None:
```

Comment:
I would use "is None".  When I do so it's mainly because it's "pythonic", but there are also reasons for doing so https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none